packages/paypal-commerce-integration/src/paypal-commerce/paypal-commerce-payment-strategy.spec.ts

```diff
@@ -207,6 +210,24 @@ describe('PayPalCommercePaymentStrategy', () => {
             });
         });
 
+        it('does not create order if orderId already defined', async () => {
+            paymentMethod.initializationData.orderId = '1';
+            await strategy.initialize(initializationOptions);
+
+            expect(paypalSdk.Buttons).toHaveBeenCalledWith({
+                fundingSource: paypalSdk.FUNDING.PAYPAL,
+                style: {
+                    color: 'black',
+                    height: 55,
+                    label: 'pay',
+                },
+                onClick: expect.any(Function),
+                onApprove: expect.any(Function),
+                onCancel: expect.any(Function),
+                onError: expect.any(Function),
+            });
+        });
```

Comment:
I don't think that it is a proper test for this label... Please, change the label or a test body to match what you want to test.

Reply:
renamed